apollo-runtime/src/androidTest/java/com/apollographql/android/cache/normalized/sql/SqlStoreTest.java

```diff
@@ -16,15 +17,15 @@
 public class SqlStoreTest {
 
   public static final String KEY = "key";
-  public static final String FIELDS = "fields";
+  public static final String FIELDS = "{\"fieldKey\": \"value\"}";
   public static final String IN_MEMORY_DB = null; //null means db is memory only
   private SqlStore sqlStore;
 
   @Before
   public void setUp() {
     ApolloSqlHelper apolloSqlHelper = ApolloSqlHelper.create(InstrumentationRegistry.getTargetContext(),
         IN_MEMORY_DB);
-    sqlStore = SqlStore.create(apolloSqlHelper, FieldsAdapter.create(new Moshi.Builder().build()));
+    sqlStore = SqlStore.create(apolloSqlHelper, FieldsAdapter.create());
```

Comment:
any reason for this change? I'd prefer to be able to use 1 moshi instance for whole app apollo or otherwise